concent_api/core/tasks.py

```diff
@@ -144,6 +149,9 @@ def verification_result(
         )
         return
 
+    if subtask.state_enum in Subtask.ACTIVE_STATES and subtask.next_deadline <= timezone.now():
```

Comment:
please use `get_current_utc_timestamp()`

Reply:
BTW: it should not be there - the handling is being done few lines below

Reply:
right, removed